### PR TITLE
chore(refs T19759): Use the new way to init Vue3

### PR DESCRIPTION
**Pull request by salisdemos**
**Ticket:** https://yaits.demos-deutschland.de/T19759

Use the new way to init Vue3
https://v3-migration.vuejs.org/migration-build.html 
Step 8. Update app entry to use [new global mounting API](https://v3-migration.vuejs.org/breaking-changes/global-api.html#a-new-global-api-createapp).

### CODE REVIEW

File: client/js/InitVue.js [hunk 1]
```diff
@@ -79,7 +57,45 @@ function initialize (components = {}, storeModules = {}, apiStoreModules = [], p
...
+    app.component('NotifyContainer', NotifyContainer)
+    app.component('DpAccordion', DpAccordion)
+    app.component('DpFlyout', DpFlyout)
+
+    Object.values(components).forEach(comp => {
+      if (comp) {
+        app.component(comp.name, comp)
+      } else {
+        console.log('at least one is undefined', components)
+      }
```
Comment (muellerdemos) [inline client/js/InitVue.js:93]:
```suggestion
    Object.keys(components).forEach(comp => {
      if (components[comp]) {
        app.component(components[comp].name, components[comp])
      } else {
        console.log(`${components[comp]} is undefined}`, components)
      }
```

Comment (muellerdemos) [reply client/js/InitVue.js:93]: Just out of curiosity, wouldn't it be beneficial if we could determine the problematic component by adjusting this logic a little bit like suggested?

**Review by muellerdemos — CHANGES_REQUESTED** [review]
Nice, config looks legit to me, however to let the build finish without errors I had to adjust `client/js/components/statement/assessmentTable/DpFilterModal.vue`:58

from: `id="userFilterSets"v-model="selectedUserFilterSet"` to:
`id="userFilterSets"
v-model="selectedUserFilterSet"`

This error is also present in the target branch `f_vue3_compat_mode_setup` hence I suggest to adjust this here.

File: client/js/components/statement/assessmentTable/DpFilterModal.vue [hunk 2]
```diff
@@ -187,7 +188,7 @@
         </div>
 
         <!-- Button row -->
-        <div class="text--right space-inline-s">
+        <div class="text-right space-inline-s">
```
Comment (muellerdemos) [inline client/js/components/statement/assessmentTable/DpFilterModal.vue:191]: i think this should also be
```suggestion
        <div class="text--right...
```
like in :79

**Review by muellerdemos — CHANGES_REQUESTED** [review]

**Comment by salisdemos** [comment]
> Nice, config looks legit to me, however to let the build finish without errors I had to adjust `client/js/components/statement/assessmentTable/DpFilterModal.vue`:58
> 
> from: `id="userFilterSets"v-model="selectedUserFilterSet"` to:
> 
> ```
> id="userFilterSets"
> v-model="selectedUserFilterSet"
> ```
> 
> This error is also present in the target branch `f_vue3_compat_mode_setup` hence I suggest to adjust this here.

done here 27d61e0284fccb8e840d288579f0044595ddd5d8

File: client/js/components/statement/assessmentTable/DpFilterModal.vue (hunk 2 above)
Comment (spiess-demos) [reply client/js/components/statement/assessmentTable/DpFilterModal.vue:190]: It is `text-right`, as the respective utility classes [have been moved to tailwind](https://github.com/demos-europe/demosplan-core/pull/1544).

**Review by muellerdemos — APPROVED** [review]
concerns were addressed and this lgtm

**Pull request merged**